tools/devops/automation/templates/build/upload-azure.yml

```diff
@@ -21,7 +21,7 @@ steps:
     allowFailedBuilds: true
     path: $(Build.SourcesDirectory)/artifacts
 
-- powershell : |
+- pwsh: |
```

Comment:
These mean the same thing right?

Reply:
Oh wait I see this 
`powershell runs Windows PowerShell and will only work on a Windows agent. pwsh runs PowerShell Core, which must be installed on the agent or container.`
Can you explain to me the difference?